packages/web-components/fast-components/src/color/recipes/neutral-stroke-divider.ts

```diff
@@ -3,14 +3,14 @@ import { Palette } from "../palette";
 import { directionByIsDark } from "../utilities/direction-by-is-dark";
 
 /**
- * The neutralDivider color recipe
+ * The neutralStrokeDivider color recipe
  * @param palette - The palette to operate on
  * @param reference - The reference color
  * @param delta - The offset from the reference
  *
  * @internal
  */
-export function neutralDivider(
+export function neutralStrokeDivider(
```

Comment:
I think this broke the color explorer - do we need to change it anywhere else in addition to there?

Reply:
That's still not breaking in a local build of the whole repo. I must not be running the same thing as CI. Fixed. Color Explorer is cheating a bit, I don't think there's anywhere else in this repo doing that.